### ISSUE TITLE
ASL Signbank upload problems

### ISSUE THREAD

**Issue by Woseseltops** [Signbank/Global-signbank#763]
This issue needs more investigation, but I'm putting it here already so we don't forget.

Julie reports video upload problems for the ASL Signbank for really specific glosses:
* JIGSAW
* FORTY-FIVE
* TWENTY-ONE
* ARMS
  
Also some images have a similar problem.

**Comment by susanodd** [comment]
See #734

**Comment by susanodd** [comment]
I have no idea whether this is related, but I'll mention it anyway just in case.
I'm working on the frequency stuff for EAF files. I'm needing to do trial runs using the test database (which starts out empty), because otherwise I can't make "data" since it creates a bunch of stuff in the database. So in my tests, I need to fill the test database with test Glosses to match the input EAF file gloss ids in my sample EAF file.

(Because users have no control over gloss ids, and these are generated by the database, it's not possible to created test data otherwise, since the gloss ids in the test file need to actually exist as glosses.

So that's the background information for the next thing.
In doing the tests, because I was creating glosses, for some reason, the tests starting actually doing stuff with video files.
It was looking for folders of the first two characters.)

I had to change the video models parameter` move_files_on_disk` to be `False`, otherwise it was "doing stuff" and generating errors.
https://github.com/Signbank/Global-signbank/blob/94dbe93ee7552b7d9cd19416ee29b86d5d18dd61/signbank/video/models.py#L380
I have no idea why the video code was being executed. But apparently there is sufficient residual data in the test database that the software thinks it has to move stuff around on disk when creating new (test) glosses. After I made that setting be False it stopped doing that.

It's possible the mysterious upload problems could be related to django administration, or that the Annotation ID has been changed, causing the folder of the gloss where the video should be stored to need to be changed? Above with the original True setting it was causing Permission Denied errors.
